cluster/images/obs-csi-plugin/socket-server.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"k8s.io/klog"
+	"net"
+	"net/http"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+const (
+	socketPath    = "/etc/csi-tool/connector.sock"
+	passwdFilePre = "passwd_file="
+	mountCmd      = "/mount"
+)
+
+type ResponseBody struct {
+	Data string
+}
+
+type MyHandler struct{}
+
+func (myHandler *MyHandler) ServeHTTP(writer http.ResponseWriter, request *http.Request) {
+	body, err := ioutil.ReadAll(request.Body)
+	if err != nil {
+		generateResponseBody(writer, http.StatusBadRequest, err.Error())
+	}
+	if len(body) == 0 {
+		generateResponseBody(writer, http.StatusBadRequest, "Request body is empty")
+	}
+	value := string(body)
+	path := request.URL.Path
+	klog.Infof("ServerHTTP Request body: %s， Request path: %s", value, path)
+
+	if path == mountCmd {
+		if err = mountHandler(value); err != nil {
+			generateResponseBody(writer, http.StatusInternalServerError, err.Error())
+		}
+		generateResponseBody(writer, http.StatusOK, "success")
+		return
+	}
+	generateResponseBody(writer, http.StatusInternalServerError, fmt.Sprintf("error request path: %s", path))
+}
+
+func generateResponseBody(writer http.ResponseWriter, statusCode int, data string) {
+	responseBody := ResponseBody{
+		Data: data,
+	}
+	response, _ := json.Marshal(responseBody)
+	writer.WriteHeader(statusCode)
+	writer.Header().Set("Content-Type", "application/json")
+	writer.Write(response) //nolint:errcheck
+}
+
+func main() {
+	flag.Parse()
+	if IsFileExisting(socketPath) {
+		if err := os.Remove(socketPath); err != nil {
+			klog.Fatalf("Failed to remove path: %s, err: %v", socketPath, err)
+		}
+	}
+
+	listen, err := net.Listen("unix", socketPath)
+	if err != nil {
+		klog.Fatalf("net.Listen failed. path: %s, err: %v", socketPath, err)
+	}
+	klog.Infof("Success start listener")
+	server := http.Server{Handler: &MyHandler{}}
+	if err := server.Serve(listen); err != nil {
+		klog.Fatalf("Socket Listener failed to close, err: %v", err)
+	}
+}
+
+func IsFileExisting(filename string) bool {
+	_, err := os.Stat(filename)
+	if err == nil {
+		return true
+	}
+	if os.IsNotExist(err) {
+		return false
+	}
+	return true
+}
+
+func mountHandler(cmd string) error {
+	if _, err := run(cmd); err != nil {
+		klog.Errorf("Failed to mount cmd: %s, err: %v", cmd, err)
+		return err
+	}
+	klog.Infof("Success to mount cmd: %s", cmd)
+	commands := strings.Split(cmd, " ")
+	for _, v := range commands {
```

Comment:
Parsing `commands` is not a good way, it is recommended to customize a message protocol.

Reply:
it will modified in another PR.